src/MCPServer/lib/archivematicaMCP.py

```diff
@@ -260,6 +260,74 @@ def cleanupOldDbEntriesOnNewRun():
     Task.objects.filter(exitcode=None).update(exitcode=-1, stderror="MCP shut down while processing.")
 
 
+def created_shared_directory_struture():
```

Comment:
This is moving responsibility for creating the sharedDirectory structure from the os installer (or ansible script) into the dashboard web installer, correct?

Shouldn't you also delete this directory:
https://github.com/artefactual/archivematica/tree/qa/1.x/src/MCPServer/share/sharedDirectoryStructure

and remove a line from:
https://github.com/artefactual/archivematica/blob/qa/1.x/src/MCPServer/debian/archivematica-mcp-server.install#L7

since that line tells the ubuntu package manager to create the shared directories, and now the dashboard is going to do it?